nuget/runners/nunit.console-runner-with-extensions.nuspec

```diff
@@ -26,14 +26,14 @@
     <releaseNotes>This release uses the latest version of the TeamCityEventListener extension.</releaseNotes>
     <language>en-US</language>
     <tags>nunit test testing tdd runner</tags>
-    <copyright>Copyright (c) 2016 Charlie Poole</copyright>
+    <copyright>Copyright (c) 2017 Charlie Poole</copyright>
     <dependencies>
       <group>
-        <dependency id="NUnit.ConsoleRunner" version="$version$" />
-        <dependency id="NUnit.Extension.NUnitProjectLoader" version="$version$" />
-        <dependency id="NUnit.Extension.VSProjectLoader" version="$version$" />
-        <dependency id="NUnit.Extension.NUnitV2ResultWriter" version="$version$" />
-        <dependency id="NUnit.Extension.NUnitV2Driver" version="$version$" />
+          <dependency id="NUnit.ConsoleRunner" version="3.5.0" />
```

Comment:
`$version$` comes from build.cake, where the version is normally advanced directly after each release - so it's currently set to 3.6. 🙂 (Which the NUnit.ConsoleRunner package should also be - that didn't even register with me last night!)